paprika/build.py

```diff
@@ -5,15 +5,44 @@
 import logging as log
 
 
+def read_tleaplines(tleapfile, pdbfile=None, skip_solvate=True):
+    """
+    Read a functional tleap file and return a list containing each line of instruction
+    
+    tleapfile : A functioning tleap input file which properly builds your system
+
+    pdbfile : If specified, replace the loadpdb target with that specified by pdbfile. Default=None.
+
+    skip_solvate : If True, skip over solvatebox or addion type statements. Default=True.
+    """
+
+    # Read tleapfile
+    tleaplines = []
+    with open(tleapfile, 'r') as f:
+        for line in f.readlines():
+            if re.search('loadpdb', line):
+                words = line.rstrip().replace('=', ' ').split()
+                if pdbfile is None:
+                    pdbfile = words[2]
+                unit = words[0]
+                tleaplines.append(
+                    "{} = loadpdb {}\n".format(unit, pdbfile))
+            if not re.search(r"^\s*addions|^\s*addions2|^\s*addionsrand|^\s*desc|"
```

Comment:
Do you think this will work better than, say, calling `grep`?

Reply:
yeah, and as mentioned previously, we were doing this already.  Now it has its own function.